src/main/java/Bro.java

```diff
@@ -0,0 +1,110 @@
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+public class Bro {
+    public static final String HORIZONTAL_LINE = "\n───────────────────────────────────────────────────────────────\n";
+    public static final String GREETING = " Sup bro. I'm Bro.\n" + " What do you want?";
+    public static final String TASK_DOES_NOT_EXIST = " Bro that task number does not exist...";
+
+    public static void main(String[] args) {
+        System.out.println(HORIZONTAL_LINE + GREETING + HORIZONTAL_LINE);
+
+        // User Input
+        String line;
+        StringBuilder reply = new StringBuilder();  // Use StringBuilder as we concatenate Strings in a loop later on
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();  // Dynamic array to store text entered by user
+        boolean endInput = false;
+        while (!endInput) {
+            line = in.nextLine();
+            String[] words = line.split(" ");
+            switch (words[0]) {
+            case "bye":
+                reply = new StringBuilder(" Bye bye bro.");
+                endInput = true;
+                break;
+            case "list":
+                reply = new StringBuilder(" Your tasks:\n");
+                for (int i = 0; i < tasks.size(); ++i) {
+                    Task task = tasks.get(i);
```

Comment:
Perhaps use currentTask instead of task as it is confusing between the object task and class Task.